client/faucet/src/main.rs

```diff
@@ -18,7 +18,7 @@ struct Args {
     #[structopt(short = "a", long, default_value = "127.0.0.1")]
     pub address: String,
     /// Faucet service listen port
-    #[structopt(short = "p", long, default_value = "80")]
+    #[structopt(short = "p", long, default_value = "8000")]
```

Comment:
80 is probably a bad default, changed to 8000 which is default port we use in our docker compose example.